src/Utils.js

```diff
@@ -86,5 +86,11 @@ function extend(...toMerge:Object) {
     return merged;
 }
 
+function focusByQuerySelector (querySelector: string) {
```

Comment:
Thanks very much for adding this function. It's really a good one to have.

Minor comment: could you rename the parameter to `selectors` please to match the parameter name of https://developer.mozilla.org/en-US/docs/Web/API/Document/querySelector to make it more transparent that it's directly using `Document.querySelector()`.